app-config.yaml

```diff
@@ -183,7 +183,16 @@ catalog:
       target: https://github.com/backstage/backstage/blob/master/plugins/techdocs-backend/examples/documented-component/documented-component.yaml
     # Backstage example APIs
     - type: url
-      target: https://github.com/backstage/backstage/blob/master/packages/catalog-model/examples/all-apis.yaml
+      target: https://github.com/backstage/backstage/blob/master/packages/catalog-model/examples/all-resources.yaml
+    # Backstage example resources
+    - type: url
+      target: https://github.com/backstage/backstage/blob/master/packages/catalog-model/examples/all-systems.yaml
+    # Backstage example systems
+    - type: url
+      target: https://github.com/backstage/backstage/blob/master/packages/catalog-model/examples/all-domains.yaml
+    # Backstage example domains
+    - type: url
+      target: https://github.com/backstage/backstage/blob/master/packages/catalog-model/examples/all-apis.yaml  
```

Comment:
I wonder why we don't use relative path here, that would make local development easier? Like:


```yaml
    - type: file
      target: ../catalog-model/examples/all-domains.yaml
```

Reply:
This already works, I use it in my `app-config.local.yaml` and we also use it in our app for testing.

Reply:
Hmmm. Sounds like a PR is coming up then! :) If people build and run in docker and other more esoteric situations though, it'll break.

Reply:
@freben Btw, was that a request to created that PR? 😆 In that case I do it after this one to avoid conflicts.